support/src/main/resources/org/jenkinsci/plugins/workflow/support/steps/StageStep/help.html

```diff
@@ -0,0 +1,6 @@
+<div>
+    By default, flow builds can run concurrently. 
+    The stage command lets you mark certain sections of a build as being constrained by limited concurrency.
```

Comment:
Most stages are used just for display purposes and do not specify `concurrency`.